mobify/progressive-web-scaffold
# Refactor Home & Skeletons

Refactor the home page's skeletons (and a few other things) so the page loads more seamlessly.

## Changes
- Refactor home container and partials
- Clean up spacing and dimensions so when skeletons disappear, the page doesn't shift
- Clean up the home page skeletons to appear less busy (closer to the final content)
- Tweak the home page CSS and markup a bit
- Change the carousel theme selectors to use `pw-` namespace instead of `c-`

| Before | After |
| ------ | ------ |
| ![skeletongs-before](https://cloud.githubusercontent.com/assets/734535/20585785/af783fe2-b1b2-11e6-82e4-0e637256293e.gif) | ![skeletongs-after](https://cloud.githubusercontent.com/assets/734535/20585789/b3e498fa-b1b2-11e6-90a5-77a1a97a0d8b.gif) |

> Notice how the content below the carousel shifts down after the content loads!

## How to test-drive this PR
- Run `npm run dev`
- [Preview Merlin's Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- On the homepage, confirm that the skeletons to real-content transition is smooth
